app/components/features/target-spf-item/component.tsx

```diff
@@ -50,55 +66,101 @@ export const TargetSPFItem: React.FC<TargetSPFItemProps> = ({
         })}
       />
       <div className="flex items-start justify-between pb-2 pr-2">
-        <span className="pr-5 text-sm font-medium font-heading">{isAllTargets ? 'Set target and SPF in all features' : name}</span>
-        {!isAllTargets && (
-          <Button
-            className="flex-shrink-0 text-xs"
-            theme="secondary"
-            size="xs"
-            onClick={() => onRemove && onRemove(id)}
-          >
-            Remove
-          </Button>
-        )}
-      </div>
-      <div className="flex">
-        <div className="relative flex-col w-full pr-4">
-          <Label ref={sliderLabelRef} className="mb-1 uppercase">
-            <span>{isAllTargets ? 'ALL TARGETS' : 'TARGET'}</span>
-          </Label>
-          <Slider
-            labelRef={sliderLabelRef}
-            minValue={0}
-            maxValue={1}
-            value={targetValue}
-            step={0.01}
-            onChange={(sliderValue) => {
-              setTargetValue(sliderValue);
-              if (onChangeTarget) onChangeTarget(+(sliderValue * 100).toFixed(0));
-            }}
-          />
-          <div className="flex justify-between w-full text-gray-400">
-            <span>0%</span>
-            <span>100%</span>
+        <div className="flex flex-col flex-grow">
+          <div className="w-full pr-5 text-sm font-medium font-heading">
+            {isAllTargets ? 'Set target and SPF in all features' : name}
           </div>
-        </div>
-        <div className="flex flex-col justify-between w-24 px-4 border-l">
-          <span>{isAllTargets ? 'ALL SPF' : 'SPF'}</span>
-          <div className="w-10 mb-6">
-            <Input
-              className="px-0 py-1"
-              theme="dark"
-              mode="dashed"
-              type="number"
-              value={FPFValue}
-              onChange={({ target: { value: inputValue } }) => {
-                setFPFValue(Number(inputValue));
-                if (onChangeFPF) onChangeFPF(Number(inputValue));
+          <div className="flex justify-between items-center mt-4">
+            <div className="flex items-center">
+              <Label ref={sliderLabelRef} className="uppercase w-16">
+                {isAllTargets ? 'ALL TARGETS' : 'TARGET'}
+              </Label>
+              <Input
+                className="w-16 text-center text-sm"
+                theme="dark"
+                type={targetInputFocused ? 'number' : 'text'}
```

Comment:
This shenanigan is to allow us to display the percentage with a `%` sign (eg: `70%`) when the user is not editing it, but still use it as a numerical when the user is editing it.